mobile/src/hooks/useHealthKit.tsx

```diff
@@ -0,0 +1,32 @@
+import {useState} from 'react';
+import AppleHealthKit from 'rn-apple-healthkit';
+
+const PERMS = AppleHealthKit.Constants.Permissions;
+
+const useHealthKit = () => {
```

Comment:
Should we abstract this into something more general like `getHealthData`? So that we have a more general purpose util before merging in

Reply:
done - see new commits